src/components/help-icon/help-icon.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Tooltip from '../tooltip/tooltip.jsx';
+import Icon from '../icon/icon.jsx';
+import helpSvg from '../../svgs/help.svg';
+
+/**
+ * @param direction Direction for the tooltip to fly from relative to the icon.
+ * @param id The id that the tooltip should be given. The element that this tooltip is describing <strong>MUST</strong> have <code>aria-describedby={id}</code>.
+ * @param label The label text of the icon.
+ * @param text The text inside of the tooltip. Tooltip text should be concise and kept to 1-2 short sentences.
+ */
+export default function HelpIcon({
+  direction,
+  id,
+  label,
+  text,
+}) {
+  return (
+    <Tooltip text={text} direction={direction} id={id}>
+      <Icon label={label} icon={helpSvg} />
+    </Tooltip>
+  );
+}
+
+HelpIcon.propTypes = {
+  direction: Tooltip.propTypes.direction,
```

Comment:
It is prefer to inline the documentation for the props interface instead of a large block at the top of the file. Using styleguidist comments will generate documentation on the site.

Something like:

```suggestion
  /** Direction for the tooltip to fly from relative to the icon. */
  direction: Tooltip.propTypes.direction,
```

Reply:
It's a bit weird to me that styleguidist prefers inline documentation instead of jsdoc.. jsdoc is pretty standardized

Reply:
Ok I dug a bit through how styleguidist prefers its documentation. These are still jsdoc, just not a blob at the top of the file, which is relieving. I think the IDE point might also still work, but I'm not positive about that. Regardless, I did this to make styleguidist happy.

2561428a